SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -45,6 +46,7 @@ void SpatialLoadBalanceEnforcer::AuthorityChanged(const Worker_AuthorityChangeOp
 			return;
 		}
 
+		check(VirtualWorkerTranslator != nullptr);
```

Comment:
Reason for adding this? We have a check in the constructor, which should suffice.